server/src/weaverbird/backends/sql_translator/steps/replacetext.py

```diff
@@ -0,0 +1,76 @@
+from distutils import log
```

Comment:
I think this entire file should be removed, the translator is supersed by pypika+snowflake dialect